app-libs/stf/src/stf_sgx.rs

```diff
@@ -70,6 +71,17 @@ impl Stf {
 		#[cfg(feature = "test")]
 		test_genesis_setup(&mut ext);
 
+		ext.execute_with(|| {
+			sp_io::storage::set(
+				&storage_value_key("Sudo", ENCLAVE_ACCOUNT_KEY),
+				&enclave_account.encode(),
+			);
+
+			if let Err(e) = Self::create_enclave_self_account(&enclave_account) {
+				error!("Failed to initialize the enclave signer account: {:?}", e);
+			}
+		});
```

Comment:
When we initialize the state, we set the enclave signer account and create a valid account for it (i.e. with a balance above the existential deposit)